src/quo2/components/tabs/tabs.cljs

```diff
@@ -142,7 +143,9 @@
         [rn/view {:style {:margin-top (- (dec unread-count-offset))}}
          [masked-view-wrapper
           {:fade-end-percentage (get @fading :fade-end-percentage) :fade-end? fade-end?}
-          [rn/flat-list
+          [(if in-scroll-view?
```

Comment:
@flexsurfer sorry for the late reply.

I am not sure why is this if condition used. I think we can just use `gesture/flat-list`

@ibrkhalil

Reply:
Because in the previous implementation, We had to choose between rn/flat-list and gesture-handler/flat-list I wanted to retain old behavior as much as possible for other usages.

Reply:
@ibrkhalil sorry that's not clear to me. Does using `gesture/flat-list` only break anything?